aqua/LyX/Portfile

```diff
@@ -31,16 +31,16 @@ depends_lib-append      port:enchant \
 
 use_xz              yes
 distname            lyx-${version}
+distfiles           ${distname}-2${extract.suffix}
```

Comment:
This is not correct. Just directly fix the `distname`. 

`distname            lyx-${version}-2`

If the `worksrcdir` is different, that add that line instead. 

`worksrcdir          lyx-${version}`